### ISSUE TITLE
How to convert the voxel data to the .mat file?

### ISSUE THREAD

**Issue by CRISZJ** [meetshah1995/tf-3dgan#33]
Thanks for your code. I want to run more demos and how to convert the ModelNet's file to the .mat file?

**Comment by 0xzhou** [comment]
> Thanks for your code. I want to run more demos and how to convert the ModelNet's file to the .mat file?

Hi, do you find any solution to transform the data format  into .mat file ? Thank you

**Comment by CRISZJ** [comment]
> > Thanks for your code. I want to run more demos and how to convert the ModelNet's file to the .mat file?
> 
> Hi, do you find any solution to transform the data format into .mat file ? Thank you

yes , there are some methods in the Internet that you can transform the data format into .mat file

**Comment by zjy6Archer** [comment]
> > > Thanks for your code. I want to run more demos and how to convert the ModelNet's file to the .mat file?
> > 
> > 
> > Hi, do you find any solution to transform the data format into .mat file ? Thank you
> 
> yes , there are some methods in the Internet that you can transform the data format into .mat file


Hello, can you share how you do the conversion? Now I have the data of . ply file that I want to convert to . mat file. thank you.
